Packs/AzureKubernetesServices/Integrations/AzureKubernetesServices/AzureKubernetesServices.yml

```
@@ -19,6 +19,18 @@ configuration:
   name: resource_group_name
   required: true
   type: 0
+- defaultvalue: https://login.microsoftonline.com
+  display: Azure AD endpoint
+  hidden: false
+  name: azure_ad_endpoint
+  options:
+  - https://login.microsoftonline.com
+  - https://login.microsoftonline.us
+  - https://login.microsoftonline.de
+  - https://login.chinacloudapi.cn
+  required: false
+  type: 15
+  additionalinfo: Azure AD endpoint associated with national cloud.
```

```suggestion
  additionalinfo: Azure AD endpoint associated with a national cloud.
```

Please change this in all the relevant .yml files and ping me

---

done in yml and README